passkey-types/src/webauthn/attestation.rs

```diff
@@ -569,13 +791,78 @@ pub enum ClientDataType {
     /// Serializes to the string `"webauthn.get"`
     #[serde(rename = "webauthn.get")]
     Get,
+
+    /// Serializes to the string `"payment.get"`
+    #[serde(rename = "payment.get")]
+    PaymentGet,
```

Comment:
**Question:** What is the spec/extension for this type?

Reply:
It's part of the Secure Payment Confirmation specification. Extension looks like so: 

```ts
  extensions: {
    "payment": {
      isPayment: true,
    }
  }
```

See https://www.w3.org/TR/secure-payment-confirmation/#client-extension-processing-authentication for more info

> In step 9, instead of creating a [CollectedClientData](https://www.w3.org/TR/webauthn-3/#dictdef-collectedclientdata), instead create a [CollectedClientPaymentData](https://www.w3.org/TR/secure-payment-confirmation/#dictdef-collectedclientpaymentdata) with:
[type](https://www.w3.org/TR/webauthn-3/#dom-collectedclientdata-type) set to "payment.get"